src/screens/OrderChat/components/MessageItem.tsx

```diff
@@ -0,0 +1,31 @@
+import { format, parseISO } from 'date-fns';
+import React from 'react';
+import * as S from '../styles';
+
+export const TIME_FORMAT = 'HH:mm - dd/LL/yyyy';
+
+interface MessageItemProps {
+  isImSender: boolean;
+  message: any;
+}
+
+export const MessageItem: React.FC<MessageItemProps> = ({ isImSender, message }) => (
```

Comment:
The `FC` type has some side effects, like always accepting a child as a prop.
```suggestion
export const MessageItem = ({ isImSender, message }: MessageItemProps) => (
```
Read more [here](https://fettblog.eu/typescript-react-why-i-dont-use-react-fc/)